### ISSUE TITLE
nodejs8.10 

### ISSUE THREAD

**Issue by davidbiller** [aws-amplify/amplify-cli#3162]
In my ...amplify/backend/auth/aivyusers/aivyusers-cloudformation-template.yml
There is still the NodeJS 8.1 
i installed the latest CLI and updateded the functions. 
So i get this error:

Reason: The runtime parameter of nodejs8.10 is no longer supported for creating or updating AWS Lambda functions. We recommend you use the new runtime (nodejs12.x) while creating or updating functions. (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: 259e6dcc-d7f6-4385-b98e-8b28242af3b4)

than i changed the version by myself. 
i tried now to push the changes and create some new functions (Social Login)

it stucks now here:

`
CREATE_COMPLETE OAuthCustomResourcePolicy AWS::IAM::Policy Fri Jan 10 2020 20:08:33 GMT+0100 (GMT+01:00)
⠇ Updating resources in the cloud. This may take a few minutes...

CREATE_IN_PROGRESS OAuthCustomResourceLogPolicy AWS::IAM::Policy Fri Jan 10 2020 20:08:37 GMT+0100 (GMT+01:00) Resource creation Initiated
CREATE_IN_PROGRESS OAuthCustomResourceLogPolicy AWS::IAM::Policy Fri Jan 10 2020 20:08:36 GMT+0100 (GMT+01:00)
⠇ Updating resources in the cloud. This may take a few minutes...

CREATE_COMPLETE OAuthCustomResourceLogPolicy AWS::IAM::Policy Fri Jan 10 2020 20:08:51 GMT+0100 (GMT+01:00)
⠇ Updating resources in the cloud. This may take a few minutes...

CREATE_COMPLETE    OAuthCustomResourceInputs Custom::LambdaCallout Fri Jan 10 2020 20:08:58 GMT+0100 (GMT+01:00)
CREATE_IN_PROGRESS OAuthCustomResourceInputs Custom::LambdaCallout Fri Jan 10 2020 20:08:58 GMT+0100 (GMT+01:00) Resource creation Initiated
CREATE_IN_PROGRESS OAuthCustomResourceInputs Custom::LambdaCallout Fri Jan 10 2020 20:08:54 GMT+0100 (GMT+01:00)
⠋ Updating resources in the cloud. This may take a few minutes...

`

**Comment by kaustavghosh06** [comment]
@davidbiller Can you check `amplify -v` and see which version of the CLI you're using?
Also, did you get a prompt to migrate your lambda functions and you responded yes to it?
Did it change any function runtime at all?

**Comment by davidbiller** [comment]
@kaustavghosh06 
4.12.0

all functions are version 10 yes

**Comment by davidbiller** [comment]
![Bildschirmfoto 2020-01-10 um 19 57 36](https://user-images.githubusercontent.com/7137361/72180157-dce96780-33e6-11ea-8553-2b7a469c72dc.png)

**Comment by davidbiller** [comment]
After id added the version by myself, it just stucks. I DONT get the this error anymore:
Reason: The runtime parameter of nodejs8.10 is no longer supported for creating or updating AWS Lambda functions. We recommend you use the new runtime (nodejs12.x) while creating or updating functions. (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: 259e6dcc-d7f6-4385-b98e-8b28242af3b4)

But it stucks... should i create an another bug? I am not sure if its stuck because the nodeJS Version

**Comment by kaustavghosh06** [comment]
@davidbiller You might need to make some changes to your code as well to avoid this. Could you take a look at this - https://aws-amplify.github.io/docs/cli/lambda-node-version-update

**Comment by davidbiller** [comment]
> @davidbiller You might need to make some changes to your code as well to avoid this. Could you take a look at this - https://aws-amplify.github.io/docs/cli/lambda-node-version-update

I've tested it, but I don't think that's what can help. I have also removed the PostConfirm. 

All functions have been changed to NodeJS 10. After I set the Auth functions to NodeJS 10 myself, I don't get an error message anymore. But the Deploy (Push) process stops. I have checked the console (CloudFormation). The last entry in the CLI is also the one that is shown as complete in the console. What should happen next, I don't see. So I can not say where the error comes from. 
As I said, I'm not even sure if it's really the nodejs version. 
I already have everything under develop running. Only ENV Production does not want to. 

I am testing a new ENV right now. Let's see if that works. 

I have the new Social Auth. CLI asked me once about 
Enter your Facebook App ID for your identity pool
And then enter your Facebook App ID for your OAuth flow again:
Could that be it? 
It is the same ID for me.
